roles/manage-user-access/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# python -c 'import crypt; print crypt.crypt("password", crypt.mksalt(crypt.METHOD_SHA512))'
```

Comment:
I believe you are capturing this here to give an example of how to generate the password - if so, please add some more meat to the bones around this comment... or, if you didn't mean to leave it here, just remove the line